imblearn/over_sampling/_random_over_sampler.py

```diff
@@ -76,6 +76,9 @@ class RandomOverSampler(BaseOverSampler):
 
     SMOTENC : Over-sample using SMOTE for continuous and categorical features.
 
+    SMOTEN : Over-sample using the SMOTE variable specifically for categorical
```

Comment:
SMOTEN : Over-sample using the SMOTE variant specifically for nominal features only.